lib/location/location.c

```diff
@@ -145,8 +145,10 @@ int location_request(const struct location_config *config)
 
 		if (config != NULL) {
 			/* Top level configs are given and must be taken from given config */
-			LOG_DBG("No method configuration given. "
-				"Using default method configuration.");
+
+			LOG_DBG("Storing received configuration as default.");
```

Comment:
Old statement is still the main point. We didn't get any method list, or method configuration. It's not saying we didn't get any configuration parameters as we actually got some main level parameters such as interval.
The fact we are storing the main level configuration parameters into `default_config` variable, is an implementation detail and convenience rather than changing the default configuration until the reboot so the updated comment is also misleading.

Reply:
ahah. I misunderstood then. Thanks for clarifying. Will close the PR.